Robus/src/robus.c

```diff
@@ -149,6 +149,11 @@ void Robus_SendMsg(ll_container_t *ll_container, msg_t *msg)
     uint16_t crc_val = 0xFFFF;
     // ********** Prepare the message ********************
     // Set protocol revision and source ID on the message
+    if(msg->header.size > MSG_BUFFER_SIZE)
+    {
+    	LUOS_ASSERT(0);
+    }
+
```

Comment:
By doing it that way macro can't avoid this IF during compiling.
To make it optimizable you have to host the condition on the macro : 

```
LUOS_ASSERT(msg->header.size <= MSG_BUFFER_SIZE);
```